src/routes.rs

```diff
@@ -614,14 +610,8 @@ pub async fn default_transform(
     let rest_path = req.param::<String>("**rest").unwrap();
 
     // We dont listen to pagination. We have a hard max of 250 per collection
-    let mut limit: i32 = 250;
-    let mut offset: i32 = 0;
-
-    // in we dont remove watched then we dont need to limit
```

Comment:
this can be adapted and kept. As it enables pagination (not supported when using excluded watched)